app/Http/Controllers/PriceMapController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PriceMapCreateRequest;
+use App\Http\Requests\PriceMapUpdateRequest;
+use App\Http\Resources\PriceMapResource;
+use Domain\PriceMap\Dtos\PriceMapCreateDto;
+use Domain\PriceMap\Dtos\PriceMapUpdateDto;
+use Domain\PriceMap\PriceMap;
+use Domain\PriceMap\PriceMapService;
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Http\Response as HttpResponse;
+use Illuminate\Support\Facades\Response;
+
+class PriceMapController extends Controller
```

Comment:
This controller and other files for PriceMap can be moved to `src/Domain/PriceMap`